remotespark/livyclientlib/result.py

```diff
@@ -0,0 +1,53 @@
+"""A set of classes which denote abstract "results" that are returned by the
+Livy session. A Result can represent a string of output that should be sent
+to Jupyter's stdout or stderr stream. A Result can also be a dataframe 
+which Jupyter renders directly itself. We may add more Result types later
+if there is a need. 
+
+Every Result has a "render" method which accepts an IPython kernel as its 
+input and returns the data associated with the Result that the kernel should
+display. The render method may also have the side effect of writing to stdout
+or stderr.
+
+Every result can also be converted to a string with the str function."""
+
+from .utils import write_to_iopub_socket
```

Comment:
You're right, this is something I forgot to take out before pushing. I'll remove it.